server/controllers/appointment.js

```diff
@@ -23,6 +27,93 @@ appointmentsRouter.post("/", async (request, response) => {
     response.json(savedAppointment);
 });
 
+// @desc Add Event to Google Calendar
+// @route GET /addevent
+appointmentsRouter.post("/addevent", async (request, response) => {
```

Comment:
```suggestion
appointmentsRouter.post("/addEvent", async (request, response) => {
```

recommend camelCase for better readability